core/src/main/scala/pickling/Macros.scala

```diff
@@ -188,7 +191,8 @@ trait PicklerMacros extends Macro with PickleMacros with FastTypeTagMacros {
               scala.pickling.PickleOps(out).pickleInto(b)
             )
           """)
-        } else (nonLoopyFields ++ loopyFields).flatMap(fir => {
+        }
+        else (nonLoopyFields ++ loopyFields).flatMap(fir => {
```

Comment:
The entire else clause is indented incorrectly, but fixing that was going to make the PR harder to review so I undid the change. I'll fix indentation afterwards.